packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.styles.ts

```diff
@@ -214,11 +210,12 @@ export const getCaretDownButtonStyles = memoizeFunction(
 
 export const getStyles = memoizeFunction(
   (theme: ITheme, customStyles?: Partial<IComboBoxStyles>, comboBoxOptionWidth?: string): Partial<IComboBoxStyles> => {
-    const { semanticColors, fonts, palette, effects } = theme;
+    const { semanticColors, fonts, effects } = theme;
     const ComboBoxRootBackground = semanticColors.bodyBackground;
     const ComboBoxRootTextColor = semanticColors.bodyText;
     const ComboBoxRootBorderColor = semanticColors.inputBorder;
     const ComboBoxRootBorderColorHovered = semanticColors.inputBorderHovered;
+    const ComboBoxRootBorderColorPressed = semanticColors.inputFocusBorderAlt;

```

Comment:
I honestly think now that we should simplify the naming pattern in here and remove the ComboBox prefix on all these consts and use camel casing: `rootBackground`. A step further would be also to remove the `color` word in it and wrap it with a `colors` object:
```ts
const colors = {
  rootBackground: semanticColors.bodyBackground,
  rootText: semanticColors.bodyText,
  rootBorderHovered: semanticColors.inputBorderHovered,
  ...
}
```

... and later just apply it with `colors._token_`